src/lib.rs

```diff
@@ -361,7 +394,9 @@ impl<T> Preferences for T
 /// easily use `std::fs::create_dir_all(..)`). Returns `None` if the directory cannot be determined
 /// or is not available on the current platform.
 pub fn prefs_base_dir() -> Option<PathBuf> {
-    get_data_root(AppDataType::UserConfig).ok()
+    directories::BaseDirs::new()
```

Comment:
Have you thought about implementing `ProjectDirs` instead of `BaseDirs` ? 

The main behaviour of this lib is about storing preferences of the current application, that's why I'm asking this

Reply:
Your suggestion might be a better fit ultimately but for now this is what I'm using and it has been working fine for me.